src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 0.3s;
```

Comment:
avoid using `all`. specify the properties

Reply:
Okay, noted.